enterprise/utils.py

```diff
@@ -495,40 +514,24 @@ def serialize_notification_content(
     """
     dashboard_url = None
     course_name = course_details.get('title')
+    course_path = '/courses/{course_id}/course'.format(course_id=course_id)
+    params = {}
+
     if admin_enrollment:
```

Comment:
For ref: this logic used to be in models.py until this PR
https://github.com/edx/edx-enterprise/pull/1299/files#diff-d950991245f878f58adc157895d6431af4385d522ab994aa25726e9abc2b434dL679

so other than dashboard_url being learner portal, all other behavior should be intact

Reply:
I copied some code from that PR to make my review simpler. I'm checking the new changes against this:
```
course_path = '/courses/{course_id}/course'.format(course_id=course_id)
        params = {}
        # add tap_hint if there is only one IdP attached with enterprise_customer
        if self.has_single_idp:
            params = {'tpa_hint': self.identity_providers.first().provider_id}

        elif self.has_multiple_idps and self.default_provider_idp:
            params = {'tpa_hint': self.default_provider_idp.provider_id}
        course_path = urlquote("{}?{}".format(course_path, urlencode(params)))

        lms_root_url = utils.get_configuration_value_for_site(
            self.site,
            'LMS_ROOT_URL',
            settings.LMS_ROOT_URL
        )
        destination_url = '{site}/{login_or_register}?next={course_path}'.format(
            site=lms_root_url,
            login_or_register='{login_or_register}',  # We don't know the value at this time
            course_path=course_path
        )
```

Reply:
ah ok thanks for doing this